.copr/Makefile

```
@@ -0,0 +1,9 @@
+SPECFILE := .copr/rovr.spec
+NAME := rovr
+VERSION := $(shell python3 -c 'import pathlib,tomllib;print(tomllib.loads(pathlib.Path("pyproject.toml").read_text(encoding="utf-8"))["project"]["version"])')
+TARBALL := $(NAME)-$(VERSION).tar.gz
+
+.PHONY: srpm
+srpm:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)